src/GitVersionExe/HelpWriter.cs

```diff
@@ -24,8 +24,8 @@ public static void WriteTo(Action<string> writeAction)
     /showvariable   Used in conjuntion with /output json, will output just a particular variable. 
                     eg /output json /showvariable SemVer - will output `1.2.3+beta.4`
     /l              Path to logfile.
-    /showconfig     Outputs the effective GitVersion config (defaults + custom from GitVersion.yaml) in yaml format
-    /overrideconfig Overrides GitVersion config values inline (semicolon-separated key value pairs e.g. /overrideconfig:tag-prefix=Foo)
+    /showconfig     Outputs the effective GitVersion config (defaults + custom from GitVersion.yml) in yaml format
+    /overrideconfig Overrides GitVersion config values inline (semicolon-separated key value pairs e.g. /overrideconfig tag-prefix=Foo)
```

Comment:
It would be nice to have some information about this in the [documentation on ReadTheDocs](http://gitversion.readthedocs.io/en/latest/usage/command-line/) too. Think you can fix that?